### PR TITLE
NO-SNOW: Two minor changes that address PR comments and fix code format

### PR DESCRIPTION
**Pull request by sfc-gh-tzhang**
- Address the [comment](https://github.com/snowflakedb/snowflake-ingest-java/pull/675#discussion_r1494896763) in https://github.com/snowflakedb/snowflake-ingest-java/commit/0c85b603517874946b939b5279b4cb57de58af49 and add a test
- Fix some format issues added by https://github.com/snowflakedb/snowflake-ingest-java/commit/a69e6f40f3ad2ed808098cb3186bec551f266449

### CODE REVIEW

**Comment by sfc-gh-tzhang** [comment]
@sfc-gh-tjones @sfc-gh-afedorov FYI

File: src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java [hunk 1]
```diff
@@ -231,7 +231,7 @@ public boolean equals(Object o) {
     if (this == o) return true;
     if (o == null || getClass() != o.getClass()) return false;
     FileColumnProperties that = (FileColumnProperties) o;
-    return Objects.equals(columnOrdinal,that.columnOrdinal)
+    return Objects.equals(columnOrdinal, that.columnOrdinal)
```
Comment (sfc-gh-japatel) [inline src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java:234]: My recent format.sh script suggested me these changes as well. 
ahh do you know why it passed in previous PR?

**Review by sfc-gh-japatel — APPROVED** [review]
lgtm! I will let you push first to not have my PR bloated up. 
FYI: `testFlushHelper` is failing

**Review by sfc-gh-xhuang — APPROVED** [review]

**Review by sfc-gh-tjones — APPROVED** [review]

File: src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java (hunk 1 above)
Comment (sfc-gh-tzhang) [reply src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java:234]: not really, i don't know why it passed the format gate

**Pull request merged**